docs/data/routes/docs/nextjs/page-routing/switching-to-ssr/en.md

```diff
@@ -3,4 +3,15 @@ name: switching-to-ssr
 routeTemplate: ./data/component-templates/article.yml
 title: Switching to SSR
 ---
-# Walkthrough: Switching to SSR
\ No newline at end of file
+# Walkthrough: Switching to SSR
+
+By default, the Next.js sample application uses a route optimized for SSG. However, we have also provided a working SSR route you can use: `\pages_examples\[[..path]].SSR.tsx`.
+
+This file may be used as a starter for SSR routes in Next.js hybrid (both SSR & SSG) applications, or may be used to switch the Next.js sample application to full SSR.
+
+> SSR may be desired in cases where you want [full Sitecore analytics and personalization functionality](/docs/nextjs/tracking-and-analytics/overview).
```

Comment:
This makes me think - "Does that mean SSG is the recommended approach for all other cases?" Do you think we need to clarify the benefit of both (or link to relevant 3rd party articles)?

Reply:
@anastasiya29 You're right, this spawns questions about SSG vs SSR which isn't really the purpose of this article. I think I'll remove this altogether (unless you have objections).

Reply:
@anastasiya29 I ended up replacing this with a callout which links to Next.js documentation covering SSG, SSR, hybrid.